app/javascript/packs/application.js

```diff
@@ -2,5 +2,7 @@ require("@rails/ujs").start()
 require("turbolinks").start()
 require("@rails/activestorage").start()
 require("channels")
-require('jquery')
-
+require('template/jquery.min.js')
+require('template/bootstrap.min.js')
+require('template/owl.carousel.min')
+require('template/custom')
```

Comment:
sorry what is `custom` file do?

Reply:
it's from template for carousel in book view I think